plan/optimizer.go

```diff
@@ -54,6 +55,40 @@ type logicalOptRule interface {
 	optimize(LogicalPlan, context.Context, *idAllocator) (LogicalPlan, error)
 }
 
+// OptimizeOnce allows us to reuse the compiled plan of a prepared statement in the same session.
+func OptimizeOnce(ctx context.Context, is infoschema.InfoSchema, pstmtID uint32, schemaVer int64, node ast.Node, useCache bool) (Plan, error) {
```

Comment:
s/OptimizeOnce/OptimizePreparedStmt/

Reply:
Thank you for the sugguestion. I've changed the codes as follows:

`-  p, err := plan.OptimizeOnce(e.Ctx, e.IS, e.ID, prepared.SchemaVersion, prepared.Stmt, prepared.UseCache)`
`+  p, err := plan.OptimizePreparedStmt(e.Ctx, e.IS, e.ID, prepared.SchemaVersion, prepared.Stmt, prepared.UseCache)`